### PR TITLE
add VM exception to the error log

### PR DESCRIPTION
**Pull request by winsvega**
move such tests under --all option

### CODE REVIEW

File: libethereum/Executive.cpp [hunk 1]
```diff
@@ -404,11 +404,13 @@ bool Executive::go(OnOpFunc const& _onOp)
 		catch (Exception const& _e)
 		{
 			// TODO: AUDIT: check that this can never reasonably happen. Consider what to do if it does.
+			std::cerr << "VM ERROR!" << endl;
```
Comment (chriseth) [inline libethereum/Executive.cpp]: Please exit the process (I have an open PR for that) - we should not continue in this state.

**Comment by chriseth** [comment]
Please merge as soon as tests (except for elcapitan) are green.

**Comment by winsvega** [comment]
ok. so this PR is just changing the test creation logic. (not execution or run)
The test results under this PR actually show how latest develop tests are run on Jenkins

**Comment by chriseth** [comment]
Argh, build error!

**Pull request merged**